data_diff/dbt_parser.py

```diff
@@ -483,19 +483,20 @@ def get_unique_columns(self) -> Dict[str, Set[str]]:
                     continue
 
                 model_node = manifest.nodes[uid]
-                if node.test_metadata.name == "unique":
-                    column_name: str = node.test_metadata.kwargs["column_name"]
-                    for col in self._parse_concat_pk_definition(column_name):
-                        if model_node is None or col in model_node.columns:
-                            # skip anything that is not a column.
-                            # for example, string literals used in concat
-                            # like "pk1 || '-' || pk2"
+                if node.test_metadata:
+                    if node.test_metadata.name == "unique":
+                        column_name: str = node.test_metadata.kwargs["column_name"]
+                        for col in self._parse_concat_pk_definition(column_name):
+                            if model_node is None or col in model_node.columns:
+                                # skip anything that is not a column.
+                                # for example, string literals used in concat
+                                # like "pk1 || '-' || pk2"
+                                cols_by_uid[uid].add(col)
+
+                    elif node.test_metadata.name == "unique_combination_of_columns":
```

Comment:
made this `elif` to avoid running 2 blocks of code for the same column attribute.